### PR TITLE
Change int to GLint for attrib_list parameters in TexStorageAttribs2DEXT and TexStorageAttribs3DEXT.

### LINKED ISSUE #498: Missing gl prefix in gl.xml
#491 added `TexStorageAttribs2DEXT` and `TexStorageAttribs3DEXT` to `gl.xml` without the `gl` prefix.
This should be an error, and it's weird that it isn't caught when generating headers from the xml.

### PR DESCRIPTION
**Pull request by janharaldfredriksen-arm**
Fixes #498 

I haven't included the regenerated headers in this. Let me know if I should.

### CODE REVIEW

File: xml/gl.xml [hunk 1]
```diff
@@ -24637,7 +24637,7 @@ typedef unsigned int GLhandleARB;
             <param group="SizedInternalFormat"><ptype>GLenum</ptype> <name>internalformat</name></param>
             <param><ptype>GLsizei</ptype> <name>width</name></param>
             <param><ptype>GLsizei</ptype> <name>height</name></param>
-            <param group="TexStorageAttribs">const int *<name>attrib_list</name></param>
+            <param group="TexStorageAttribs">const GLint *<name>attrib_list</name></param>
```
Comment (oddhack) [inline xml/gl.xml:24640]: Should be

> const \<ptype>GLint\</ptype>

likewise for the other change.

File: xml/gl.xml [hunk 2]
```diff
@@ -24637,7 +24637,7 @@ typedef unsigned int GLhandleARB;
             <param group="SizedInternalFormat"><ptype>GLenum</ptype> <name>internalformat</name></param>
             <param><ptype>GLsizei</ptype> <name>width</name></param>
             <param><ptype>GLsizei</ptype> <name>height</name></param>
-            <param group="TexStorageAttribs">const int *<name>attrib_list</name></param>
+            <param group="TexStorageAttribs">const <ptype>GLint</ptype>* <name>attrib_list</name></param>
         </command>
         <command>
             <proto>void <name>TexStorageAttribs3DEXT</name></proto>
```
Comment (Perksey) [inline xml/gl.xml:24643]: I think the main concern in #498 was the function names.
```suggestion
            <proto>void <name>glTexStorageAttribs3DEXT</name></proto>
```
(and elsewhere)

Comment (janharaldfredriksen-arm) [reply xml/gl.xml:24643]: yikes. thanks.

Comment (NogginBops) [reply xml/gl.xml:24643]: I also want the `int` to `GLint` change to happen as `GLint` is what is used everywhere. I didn't mention that in #498 but I did ask here about it https://github.com/KhronosGroup/OpenGL-Registry/pull/491#issuecomment-1047971998.

**Review by Perksey — APPROVED** [review]

**Comment by pdaniell-nv** [comment]
@oddhack this is approved to merge.

**Pull request merged**